hx_lti_initializer/views.py

```diff
@@ -75,6 +76,10 @@ def launch_lti(request):
             debug_printer('DEBUG - user_id not found in post.')
             raise PermissionDenied()
 
+    lti_grade_url = get_lti_value('lis_outcome_service_url', tool_provider)
+    if lti_grade_url is not None:
+        save_session(request, is_graded=True, lti_params=request.POST)
```

Comment:
It might make sense to save the `lti_params` regardless of the presence of the `lti_grade_url`. I could see that being useful in other contexts.